university-agent/src/main/java/nl/quintor/studybits/controller/BootstrapController.java

```diff
@@ -34,6 +32,9 @@
     @Autowired
     private Seeder seeder;
 
+    @Autowired
```

Comment:
Autowired should be optional (@Autowired(required = false))

Reply:
Done